components/organisms/Layout.js

```diff
@@ -33,6 +33,18 @@ export const Layout = ({
 
   return (
     <div className="overflow-x-hidden">
+      <nav className="skip-main">
+        <a
+          id="skipToMainContent"
+          className="bg-custom-blue-dark text-white py-1 px-2 hover:bg-gray-dark"
+          href="#pageMainTitle"
+          dataCyButton={"skip-Content"}
```

Comment:
Getting a warning for this attribute 
![image](https://user-images.githubusercontent.com/31868510/121022790-abf29b00-c770-11eb-87eb-71eadf13c526.png)